src/test/java/com/opipo/codechallenge/service/TransactionStatusChannelServiceTest.java

```diff
@@ -0,0 +1,100 @@
+package com.opipo.codechallenge.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import com.opipo.codechallenge.repository.model.TransactionEntity;
+import com.opipo.codechallenge.service.impl.TransactionStatusChannelServiceImpl;
+import com.opipo.web.api.model.TransactionStatusRequest.ChannelEnum;
+import com.opipo.web.api.model.TransactionStatusResponse;
+
+@DisplayName("TransactionStatusChannelService test")
+public class TransactionStatusChannelServiceTest {
+
+    private TransactionStatusChannelService transactionStatusChannelService;
+
+    @BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/albertinizao/codechallenge/pullRequest?prid=4681930)